prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -812,6 +812,28 @@ class ActionHistory : Logging {
                     }
                 }
                 jsonGen.writeEndArray()
+
+                jsonGen.writeArrayFieldStart("filteredReportRowObjects")
```

Comment:
I think this needs to be added to the submission history API?  If so, open a new issue for that.  All this is about to be replaced with those objects from submissions API.